### ISSUE TITLE
can't install sudo

### ISSUE THREAD

**Issue by bryanlarsen** [artificialio/docker-sails#10]
Your Dockerfile doesn't install sudo, but it does create a sudoers file.   This means that a subsequent apt-get install sudo will fail, even if installed with `--assume-no`.

**Comment by kriswill** [comment]
I haven't tried it, kind of busy right now, but can't you do:

``` sh
USER root
RUN apt-get install -y --no-install-recommends sudo
```

to install sudo?  also, why not just do `USER root` to do whatever `RUN apt-get` you need, and switch back to `USER sane` to start your process?

**Comment by bryanlarsen** [comment]
no, that conflicts on `/etc/sudoers`, and neither -y or --assume-no will get past it.   --force-yes probably would, but that's not a good idea.

And I want sudo when I'm doing docker exec bash, not in my Dockerfile.

**Comment by kriswill** [comment]
@Globegitter should we install `sudo`?  I'm not sure it's wise, or a reasonable default, but I can empathize with the desire to bash into your container and sudo install software.
